pkgs/by-name/su/supersonic/package.nix

```diff
@@ -35,9 +42,18 @@ buildGoModule rec {
   buildInputs = [
     libglvnd
     mpv
+  ] ++ lib.optionals stdenv.isLinux [
     xorg.libXxf86vm
     xorg.libX11
-  ] ++ (glfw3.override { inherit waylandSupport; }).buildInputs;
+  ]
+  ++ (builtins.filter (p: !(lib.hasPrefix "apple-framework" p.name))
+    (glfw3.override { inherit waylandSupport; }).buildInputs)
```

Comment:
I'd prefer the `buildInputs` to be explicit - the workaround there is terribly hacky.

Reply:
Fair point, I'm not too fussed either way. I see you've already changed it 👍